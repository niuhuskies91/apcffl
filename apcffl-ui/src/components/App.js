import React from 'react';

const App = () => {
  return (
    <section>
      <h1>Frigg off, Barb!</h1>
    </section>
  );
}

export default App
