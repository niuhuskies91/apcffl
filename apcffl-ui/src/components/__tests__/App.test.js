import React from 'react';

import App from '../App';

describe('App component', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = render(<App />);
  });

  it('renders', () => {
    const { asFragment } = wrapper;
    expect(asFragment()).toMatchSnapshot();
  });
});
