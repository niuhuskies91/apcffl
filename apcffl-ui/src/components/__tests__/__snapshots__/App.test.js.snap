// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component renders 1`] = `
<DocumentFragment>
  <section>
    <h1>
      Frigg off, Barb!
    </h1>
  </section>
</DocumentFragment>
`;
